test/source/tests/browser-unit-tests/unit-Catch.js

```diff
@@ -48,3 +48,16 @@ BROWSER_UNIT_TEST_NAME(`Catcher does report on consumer`).consumer;
   }
   return "pass";
 })();
+
+BROWSER_UNIT_TEST_NAME(`Catcher does report sensitive infos`);
+(async () => {
+  const url = 'chrome://extension-id/pgp_block.htm?frameId=id&message=blahblah&senderEmail=blahblah&acctEmail=blahblah';
+  const censoredUrl = Catch.censoredUrl(url);
+  const urlParams = Url.parse(['message', 'senderEmail', 'acctEmail'], censoredUrl);
```

Comment:
Why don't we just compare `censoredUrl` against the expected string?
`chrome://extension-id/pgp_block.htm?frameId=id&message=[SCRUBBED]&senderEmail=[SCRUBBED]&acctEmail=[SCRUBBED]`
This test doesn't show that `frameId` remains not scrubbed.

Reply:
It'd be also nice to add more edge test cases with scrubbed parameters interspersed with non-scrubbed. e.g.
`chrome://extension-id/pgp_block.htm?frameId=id&message=blahblah&some=1&senderEmail=blahblah&bbb=123`
that make sure our regex is acting in greedy mode and leaves `frameId`, `some` and `bbb` unscrubbed

Reply:
Trying to display which params didn't exactly scrubbed. But comparing against expected string also looks okay to me.
> Why don't we just compare censoredUrl against the expected string?